config/routes.rb

```diff
@@ -128,4 +128,12 @@
   end
 
   get :healthcheck, to: proc { [200, {}, %w[OK]] }
+
+  resources :local_authorities, only: %i[edit update]
+
+  get(
+    :administrator_dashboard,
+    action: :show,
+    controller: :administrator_dashboard
+  )
```

Comment:
why not use resources here ?

Reply:
To avoid having to use an id! https://guides.rubyonrails.org/routing.html#singular-resources

There's probably more than one way to do this though 🤔

Reply:
Ah I think I ran into this issue before and I ended up doing
https://github.com/unboxed/bops/blob/main/config/routes.rb#L107-L110

But if the docs say to do as you did then no issues with me

Reply:
you can use the word `resource` singular

so it can be:
```
resource : administrator_dashboard, only: :show
```

Reply:
The singular `resource`, will do the trick without referencing an ID and that's why it doesn't have an `index` action

Reply:
aha...nice, thanks both!